inc/layers.php

```diff
@@ -122,25 +123,29 @@ function post_layers_box($post = false) {
       <table class="layers-list available-layers">
        <tbody data-bind="foreach: filteredLayers">
         <tr>
-         <td><strong data-bind="text: title"></strong></td>
+         <td><strong><a data-bind="text: title, attr: {href: url}" target="_new"></a></strong></td>
```

Comment:
@Huyeng **target** attributes of <a> cannot have the value "_new" (https://www.w3schools.com/tags/att_a_target.asp). Did you maybe mean "_blank" here?